src/accelerate/data_loader.py

```diff
@@ -354,11 +356,12 @@ class DataLoaderShard(DataLoader):
         - **total_dataset_length** (`int`) -- Total length of the inner dataset across all processes.
     """
 
-    def __init__(self, dataset, device=None, rng_types=None, synchronized_generator=None, **kwargs):
+    def __init__(self, dataset, device=None, rng_types=None, synchronized_generator=None, skip_batches=0, **kwargs):
```

Comment:
```suggestion
    def __init__(self, dataset, device=None, rng_types=None, synchronized_generator=None, skip_n_batches=0, **kwargs):
```

Suggestion based on previous comment